examples/scripts/parameter_file.txt

```diff
@@ -10,8 +10,7 @@ ny=256
 # number of grid cells in the z dimension
```

Comment:
If this is just an example input parameter file for running the 3D disk test, I recommend renaming it something (like "disk" and moving it to the examples directory.

Reply:
Yep